src/styles/components/_related-components.scss

```diff
@@ -1,4 +1,4 @@
-$breakpoint-phone: 600px;
+$breakpoint-phone: 900px;
```

Comment:
900px is a very large screen width to consider a mobile breakpoint, maybe include a tablet breakpoint and then also add a variable for the mobile breakpoint?